schemas/2017-06-01/Microsoft.StorSimple.8000.json

```diff
@@ -0,0 +1,1699 @@
+{
+  "id": "https://schema.management.azure.com/schemas/2017-06-01/Microsoft.StorSimple.8000.json#",
```

Comment:
Looks like we already have a definition for this resource type here: https://github.com/Azure/azure-resource-manager-schemas/blob/master/schemas/2017-06-01/Microsoft.StorSimple.json.

Please could you remove this to avoid duplication?